packages/tables/resources/views/components/actions/action.blade.php

```diff
@@ -19,14 +19,14 @@
     :dark-mode="config('tables.dark_mode')"
     :attributes="\Filament\Support\prepare_inherited_attributes($attributes)->merge($action->getExtraAttributes())"
     :tag="$action->getUrl() ? 'a' : 'button'"
-    :wire:click="$action->isEnabled() ? $wireClickAction : null"
     :href="$action->isEnabled() ? $action->getUrl() : null"
     :target="$action->shouldOpenUrlInNewTab() ? '_blank' : null"
     :disabled="$action->isDisabled()"
     :color="$action->getColor()"
     :tooltip="$action->getTooltip()"
     :icon="$icon ?? $action->getIcon()"
     :size="$action->getSize() ?? 'sm'"
+    x-on:click="{!! $action->isEnabled() ? $wireClickAction : null !!}"
```

Comment:
replaced `wire:click` with `x-on:click` because these actions will be teleported outside the Livewire component using Alpine `x-teleport`. So wire:click will not be available anymore.

Now the alpine component calls the `mountTableAction` function.